kitchen.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Mondoo, Inc.
```

Comment:
Chef uses a `Copyright::` format in headers. Cookstyle will enforce it.

Source: I wrote the cop and did the historical digging into why Adam used this format.

```suggestion
# Copyright:: (c) Mondoo, Inc.
```

Reply:
Fix coming...

Reply:
pushed the change

Reply:
They all need to get updated